### PR TITLE
add gpg content

### PR DESCRIPTION
**Pull request by beardofedu**
This pull request adds GPG content to the training manual as identified in #240.

### CODE REVIEW

**Comment by haothang** [comment]
240

**Review by rwnfoo — DISMISSED** [review]

File: docs/24_commit_sig_verification.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+## Commit Signature Verification
+
+Verifying the authenticity of the source of commits submitted to your repository might be important to you and your team. GitHub supports methods of ensuring that the author of the commit in a pull request was actually created from a trusted source. Commit signature verification is a cryptographically verifiable method of ensuring that commits coming from Mona Lisa are in fact coming from Mona Lisa.
+
+Signed commits can be created by implementing GPG or S/MIME signature that is attached to every commit that you make. Once you have integrated GPG or S/MIME on your machine and account, commits you create will appear like this in the GitHub UI:
```
Comment (ppremk) [inline docs/24_commit_sig_verification.md:5]:
```suggestion
Signed commits can be created by implementing GPG or S/MIME signature that is attached to every commit that you make on your local machine (downstream repository). Once you have integrated GPG or S/MIME on your machine and account, commits you create locally and push to the upstream will appear like this in the GitHub UI:
```

File: docs/24_commit_sig_verification.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+![Signed commit example](https://docs.github.com/assets/images/help/commits/verified-commit.png)
+
+An unverified commit will not display the "Verified" tag in the GitHub interface.
```
Comment (ppremk) [inline docs/24_commit_sig_verification.md:9]:
```suggestion
An unsigned commit on the local downstream repo will not display the "Verified" tag in the GitHub interface. Any commits made directly on the GitHub interface will automatically include the "Verified" tag.
```

File: docs/24_commit_sig_verification.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Create a commit and push your changes by entering the following:
+
+    ```bash
+    git add README.md
+    git commit -S -m  "Add author to README"
+    git push -u origin add-author
+    ```
+
+1. Create a pull request and in the **Commits** tab you will see the **Verified** tag next to your commit.
```
Comment (ppremk) [inline docs/24_commit_sig_verification.md:100]:
```suggestion
1. Create a pull request and in the **Commits** tab you will see the **Verified** tag next to your commit.


> Note: If you've enforced branch protections in your repository, you can set up required commit signing. When you enable required commit signing on a branch, contributors can only push local commits to the branch if they are signed and verified. Contributors can merge signed and verified commits using the GitHub Enterprise web interface. For more information, see [Configuring protected branches.](https://docs.github.com/en/enterprise/2.15/user/articles/configuring-protected-branches)
```

**Review by ppremk — CHANGES_REQUESTED** [review]
Added some minor tweaks @beardofedu

**Review by parkerbxyz — APPROVED** [review]

**Comment by beardofedu** [comment]
@ppremk can you re-review this since @parkerbxyz implemented your changes? Once you are done, just merge it 👍

**Review by ppremk — APPROVED** [review]

**Pull request merged**